subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c

```diff
@@ -1554,11 +1662,13 @@ int nrf_cloud_pgps_init(struct nrf_cloud_pgps_init_param *param)
 #endif
 
 	__ASSERT(param != NULL, "param must be provided");
-	__ASSERT(param->storage_base != 0u, "P-GPS flash storage must be provided");
+	//__ASSERT(param->storage_base != 0u, "P-GPS flash storage must be provided");
```

Comment:
👀

Reply:
Whoa, wait, how is compliance not angry about this?